sample/src/main/java/com/karumi/headerrecyclerview/sample/util/DebugHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.karumi.headerrecyclerview.sample.util;
+
+import android.util.Log;
+
+/**
+ * Debug utility class
+ */
+public class DebugHelper {
+  private static String getCallMethodAndLine() {
```

Comment:
We don't need this ``DebugHelper`` class. Just add a simple log on ``DragonBallAdapter`` methods :)